.github/CODEOWNERS

```diff
@@ -1,7 +1,2 @@
-# The Design System will be the default owners for everything in
-# the repository except for package.json and src directory.
-# Some very complex components will be co-owned with other teams.
-
-# Components
-packages/fondue/src/components/RichTextEditor @Frontify/guidelines-one
-packages/fondue/src/components/Tree @Frontify/guidelines-navigation-themes
```

Comment:
both GitHub Teams do not exist anymore.

Considering that the `fondue` legacy is maintained until we can migrate the components to the new package, we can avoid adding sub-rules for the time being and just consult the team if any changes are required.